src/deep-resource/lib/Resource/Action.js

```diff
@@ -21,8 +21,9 @@ export class Action {
    * @param {String} region
    * @param {Boolean} forceUserIdentity
    * @param {Object} apiCache
+   * @param {String} scope
    */
-  constructor(resource, name, type, methods, source, region, forceUserIdentity, apiCache) {
+  constructor(resource, name, type, methods, source, region, forceUserIdentity, apiCache, scope) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This function has too many parameters. (9)](https://www.codacy.com/app/MitocGroup/deep-framework/file/2420483586/issues/source?bid=2895475&fileBranchId=3238728#l26)